GDAXSharp/WebSocket/Models/Response/Level2.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace GDAXSharp.WebSocket.Models.Response
+{
+    public class Level2 : BaseMessage
+    {
+        public string ProductId { get; set; }
```

Comment:
i think this should be a `ProductType`. just add the `[JsonConverter(typeof(StringEnumConverter))]` above it